modules/react/checkbox/lib/CheckboxCheck.tsx

```diff
@@ -0,0 +1,81 @@
+import * as React from 'react';
+import {createComponent} from '@workday/canvas-kit-react/common';
+import {createStencil, createStyles, cssVar, calc, px2rem} from '@workday/canvas-kit-styling';
+import {brand, system} from '@workday/canvas-tokens-web';
+import {SystemIcon} from '@workday/canvas-kit-react/icon';
+import {checkSmallIcon} from '@workday/canvas-system-icons-web';
+import {CheckBackground} from './CheckBackground';
+
+interface CheckboxCheckProps {
+  variant?: 'inverse';
+  checked: boolean;
+  indeterminate?: boolean;
+  error?: 'error' | 'alert';
+}
+
+const checkboxCheckStencil = createStencil({
+  base: {
+    display: 'flex',
+    flexDirection: 'column',
+    maxWidth: '100%',
+    pointerEvents: 'none',
+    transition: 'transform 200ms ease, opacity 200ms ease',
+    span: {
+      marginLeft: '-6px',
+      transition: 'margin 200ms ease',
+    },
+    opacity: 0,
+    transform: 'scale(0.5)',
+  },
+  modifiers: {
+    checked: {
+      true: {
+        opacity: 1,
+        transform: 'scale(1)',
+      },
+    },
+    indeterminate: {
+      true: {
+        opacity: 1,
+        transform: 'scale(1)',
+      },
+    },
+    variant: {
+      inverse: {
+        '& > div': {
+          backgroundColor: brand.primary.base,
+        },
+      },
+    },
+  },
+});
+
+const indeterminateBoxStyles = createStyles({
+  width: calc.add(system.space.x2, px2rem(2)),
```

Comment:
`add` is also exported yeah? idk i like the object notation

Reply:
yes, Nicholas said use it with `calc` to help transformer understand it